submissions/kissik/data-structures/index.js

```diff
@@ -0,0 +1,110 @@
+const http = require('http');
+
+const fs = require('fs');
+
+const port = process.env.PORT || 3210;
+
+const list = {};
+
+list.data = {};
+
+list.push = data => {
+  list.data = {
+    item: data,
+    next: list.data
+  };
+};
+
+list.pop = () => {
+  const a = list.data.item ? list.data.item : 'none';
+  if (list.data) list.data = list.data.next;
+  return a;
+};
+
+list.print = msg => {
+  let a = list.data;
```

Comment:
What for?

Reply:
for list loop

Reply:
Sorry, it was too abstract question :)

Can you rename it? I mean, something like `listItem`?

Reply:
ok))))
but `a` is _THE LIST_ not the `listItem` so better name, on my opinion, is `data`

Reply:
In general, `data` not a very descriptive name, because it can be hard to understand what sort of data saved in a variable with such a naming.

Following the rule of thumb, `listContent` or `listData` will be better.

But this is a small function so `const { data } = list` can also be good.

The choice is up to you. 😄